src/selectors/linesInScope.js

```diff
@@ -0,0 +1,41 @@
+import { getOutOfScopeLocations, getSelectedSource } from "../selectors";
+
+import range from "lodash/range";
+import flatMap from "lodash/flatMap";
+import uniq from "lodash/uniq";
+import without from "lodash/without";
+
+function getLinesOutOfScope(outOfScopeLocations: AstLocation[]) {
+  if (!outOfScopeLocations) {
+    return null;
```

Comment:
... not sure. i kinda like having explicit empty states to distinguish between nothing and we happen to have no out of scope locs. ill think about it